addons/bindings.md

```diff
@@ -27,13 +27,29 @@ The following bindings can be installed and configured directly from within Pape
 {% for binding in sorted_bindings %}| [{{ binding.label }}]({{base}}/addons/bindings/{{ binding.id }}/readme.html) | {{ binding.description }} |
 {% endfor %}
 
-## Legacy 1.x Bindings
+## 1.x Bindings
 
-Bindings developed for the first version of openHAB can also be used with openHAB 2.
-Please be aware, that these bindings function and behave differently.
+Bindings developed for the first version of openHAB can also be used with openHAB 2. If there is already a 2.x binding version available, those are referred to as "Legacy 1.x Bindings", otherwise as regular add-ons.
+Please be aware, that these bindings function and behave differently, f.e. there is no autodiscovery function for openHAB2 things and openHAB2 channels are not available.
 For beginners it is recommended to resort to native bindings if possible.
 
-To be able to see and install 1.x bindings needs to be activated in the system settings.
+To be able to see and install 1.x bindings one has to activate "Access Remote Repository" and "Include Legacy 1.x Bindings" in openHAB2 system settings.
```

Comment:
Okay, found the source:

https://community.openhab.org/t/a-single-distro-from-now-on/19903?u=sihui

> For the ones that loved the offline distro (like myself): There is an easy way to get it back! Just additionally download the file openhab-addons-2.0.0-SNAPSHOT.kar and place it in your local addons/ folder - this will immediately make all add-ons available offline and you can safely deactivate the online repo access (in addons.cfg or through the Paper UI). You can even do the same now with the legacy add-ons (which were only available online before) - simply download the openhab-addons-legacy-2.0.0-SNAPSHOT.kar and put it into the addons/ folder.

For me that means, if you don't download the *.kar, you have to enable both options.